src/_includes/head.html

```diff
@@ -1,24 +1,43 @@
-<title>{% if page.title %}{{ page.title }}{% else %}Software Craftsmanship and Agile Development{% endif %} | Codurance | Craft at Heart | London | Barcelona</title>
+<title>
+    {% if page.url == "/software-modernisation/" and site.lang == "es" %}
+        {{ page.title-es }}
+    {% elsif page.url == "/software-modernisation/" and site.lang != "es" %}
+        {{ page.title }}
+    {% else %}
+        {% if page.title %}
+            {{ page.title }}
+        {% else %}
+            Software Craftsmanship and Agile Development
+        {% endif %}
+        | Codurance | Craft at Heart | London | Barcelona
+    {% endif %}
+</title>
+
  <!-- Meta -->
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="description" content="Our team of dedicated software craftspeople provide consultancy, software development, and training services to clients seeking high quality development processes and software solutions.">
+<meta name="description" content="{% if page.url == '/software-modernisation/' and site.lang == 'es' %}{{ page.description-es }}{% elsif page.url == '/software-modernisation/' and site.lang != 'es' %}{{ page.description }}{% else %}Our team of dedicated software craftspeople provide consultancy, software development, and training services to clients seeking high quality development processes and software solutions.{% endif %}" />
 <meta name="author" content="Codurance Ltd">
```

Comment:
Same deal here. It feels like this logic should be inside `page.description` logic rather than spread into the HTML as well.

Reply:
Yeah ideally we should just have a variable for the result of the if, injected into the meta.